<launch>
    <rosparam command="load" file="$(find ig_lio)/config/ntu1.yaml" />

    <node pkg="ig_lio" type="ig_lio_node" name="ig_lio_node" output="screen" />
    
</launch>